bin/generate-shell-css.js

```diff
@@ -0,0 +1,25 @@
+var http = require( 'http' );
+var critical = require( 'critical' );
+
+// Load the HTML page from the dev server. This will fail if the dev server is
```

Comment:
I think it's worth a try. Maybe in the `postbuild` step when the big CSS bundle is already generated. I don't consider iti a blocker for this PR though, just a "nice to have" :)